project-addons/custom_partner/models/partner.py

```diff
@@ -713,3 +715,18 @@ class ProductSupplierInfo(models.Model):
     _inherit = 'product.supplierinfo'
 
     ref_supplier = fields.Char("Ref. Supplier", related='name.ref_supplier', readonly=True)
+
+
+class Followers(models.Model):
+    _inherit = 'mail.followers'
+
+    @api.model
+    def create(self, vals):
```

Comment:
¿Qué hacéis aquí? ¿es como si estuvieran duplicados y lo evitas no? ¿en que modelos pasa? porque tiene pinta de bug que podemos corregir y no hacer esto que ralentiza toda creación de seguidores.

Reply:
Eso es, al guardar un pedido salta el error de la constraint que impide que la misma persona siga varias veces un mismo modelo. Por lo que he probado sólo pasa en res.partner. Es cierto que el parche no me convence demasiado, pero he intentado buscar donde añadíamos en algún módulo nuestro los seguidores y no he dado con ello

Reply:
OK, lo apruebo para que no falle, pero dejmaos pendiente darle una vuelta a ver si encontramos quien provoca la duplicidad porque no debería de darse.